WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java

```diff
@@ -259,7 +262,10 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
 
         if (savedInstanceState == null) {
-            if (mIsEmailSignup) {
+             // Start loading reader tags so they will be available asap
+             ReaderUpdateServiceStarter.startService(getActivity(), EnumSet.of(ReaderUpdateLogic.UpdateTask.TAGS));
```

Comment:
This is a good idea to trigger the service to retrieve the tags as soon as possible!  There is something similar in the login process.  Is there a reason this was added to [`SignupEpilogueFragment`](https://github.com/wordpress-mobile/WordPress-Android/blob/e4e705f712b69ae1af0d761e9ccf96eb4060fdf1/WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java) rather than [`SignupEpilogueActivity`](https://github.com/wordpress-mobile/WordPress-Android/blob/e4e705f712b69ae1af0d761e9ccf96eb4060fdf1/WordPress/src/main/java/org/wordpress/android/ui/accounts/SignupEpilogueActivity.java) for signup like [`LoginActivity`](https://github.com/wordpress-mobile/WordPress-Android/blob/e4e705f712b69ae1af0d761e9ccf96eb4060fdf1/WordPress/src/main/java/org/wordpress/android/ui/accounts/c.java) for login?

Reply:
The only reason is that `SignupEpilogueActivity` only serves as a container for `SignupEpilogueFragment`, while `LoginActivity` is way more complicated, and having this logic on activity level makes more sense there, at least for me.

Reply:
I guess it's effectively the same thing.  I just wasn't sure if we wanted to keep it in the activity for consistency, but it looks like other instances have it in a fragment too.  So leaving it in `SignupEpilogueFragment` should be fine.

A couple instances of the `ReaderUpdateServiceStarter` ([`LoginActivity`](https://github.com/wordpress-mobile/WordPress-Android/blob/e4e705f712b69ae1af0d761e9ccf96eb4060fdf1/WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java#L626-L629) and [`AppSettingsFragment`](https://github.com/wordpress-mobile/WordPress-Android/blob/e4e705f712b69ae1af0d761e9ccf96eb4060fdf1/WordPress/src/main/java/org/wordpress/android/ui/prefs/AppSettingsFragment.java#L406-L407)) pass the application context (`getApplicationContext()` and `WordPress.getContext()`) since the activity may be destroyed by the time the service uses the context.  Should we do the same here since `SignupEpilogueActivity` and `SignupEpilogueFragment` may be short-lived?